xmtp_mls/src/identity_updates.rs

```diff
@@ -119,44 +164,17 @@ where
     /// If no `to_sequence_id` is provided, use the latest value in the database
     pub async fn get_association_state(
         &self,
-        conn: &DbConnection<<Db as XmtpDb>::Connection>,
+        _conn: &DbConnection<<Db as XmtpDb>::Connection>,
```

Comment:
I could remove this from all the places it's used instead of marking it as _. Just want to confirm first that this is the route we want to go.

Reply:
could just replace the call to mls_provider with dbconnection, then this would be used (see other comment as well)